src/ensembl/production/metadata/grpc/service.py

```diff
@@ -13,9 +13,20 @@
 import grpc
 import logging
 
+from ensembl.production.metadata.grpc.config import MetadataConfig as cfg
 from ensembl.production.metadata.grpc import ensembl_metadata_pb2_grpc
 from ensembl.production.metadata.grpc.servicer import EnsemblMetadataServicer
 
+logger = logging.getLogger(__name__)
+
+# Determine the logging level based on the value of cfg.debug_mode
+log_level = logging.DEBUG if cfg.debug_mode else logging.INFO
```

Comment:
use a config var ? Best is warning in production.

Reply:
Done